Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "BISC195Labs"
 uuid = "3c894fce-1a5f-4a14-a547-781e8c68175f"
-authors = ["Kevin Bonham, PhD"]
+authors = ["Kevin Bonham, PhD", "Jen Enriquez", "Emily Lu"]
```

Comment:
Only `Enriquez` in the title - but should this count as submission for both of you @jje009wells @lushaojua?

Reply:
Nevermind, I see Emily has her own PR #6